superset/db_engine_specs/presto.py

```diff
@@ -887,6 +887,10 @@ def _parse_structural_column(  # pylint: disable=too-many-locals
         Parse a row or array column
         :param result: list tracking the results
         """
+        # We remove the (key_type, result_type) annotations for maps, as these types
+        # trip up the parser, and they are not reflected in SQLAlchemy column types
+        parent_data_type = re.sub("map\(\w+,\s*\w+\)", "map", parent_data_type)
```

Comment:
I realized I had a missing `r` prefix, but can't easily amend right now.

```suggestion
        parent_data_type = re.sub(r"map\(\w+,\s*\w+\)", "map", parent_data_type)
```